core/runtime/runtime_api/impl/block_builder.cpp

```diff
@@ -30,6 +30,9 @@ namespace kagome::runtime {
   outcome::result<std::vector<primitives::Extrinsic>>
   BlockBuilderImpl::inherent_extrinsics(RuntimeEnvironment &env,
                                         const primitives::InherentData &data) {
+    // Ignore changes.
```

Comment:
Could you also comment why?

Reply:
Because this call is not persistent, [#1464 block_builder.cpp](https://github.com/soramitsu/kagome/commit/20d48f30abe7d1ebde9a9755785fb047bd2f315d#diff-650eb63f6326ea3ebe6543ae9106bf8e39813d842a7e645e210f077b130ced0eL39-L40)